content/blog/eclipse-adoptium-projects-ready-for-community-review/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Eclipse Adoptium Projects Ready for Community Review
+date: "2020-12-21T12:00:00+00:00"
+author: tsc
+description: Community Review is the last step before we can officially start Eclipse Adoptium. It gives all of you an opportunity to ask questions, suggest improvements, or raise concerns. We also have an update to our transition roadmap.
+tags:
+  - AdoptOpenJDK
+  - Eclipse
+  - Foundation
+---
+
+Six months ago, we announced that AdoptOpenJDK is going to join the Eclipse Foundation and rebrand as Eclipse Adoptium. After a long period of work behind the scenes, the first batch of project proposals is ready for community review. At the Eclipse Foundation, the monolithic AdoptOpenJDK project will be split into multiple sub-projects under the umbrella of the [Eclipse Adoptium Top Level Project](https://projects.eclipse.org/projects/adoptium):
+
+* **Eclipse AQAvit** encompasses all our activities around testing, especially the development of our [AQA test suite](/2019/07/the-first-drop-introducing-adoptopenjdk-quality-assurance-aqa-v1-0/).
+* **Eclipse Temurin** (an anagram of "runtime") focuses on building and releasing JDKs (that part that most associate with AdoptOpenJDK).
```

Comment:
Do not expect you to change the text, just wanted to include in the review for posterity...

Temurin is not only an anagram of "runtime" but a chemical compound, 1,3,7,9-tetramethyluric acid similar to caffeine... (with the bonus of being demonstratively non-habituating and clinically safe)
<img width="321" alt="Screen Shot 2020-12-21 at 12 15 48 PM" src="https://user-images.githubusercontent.com/2836948/102803812-d6999480-4386-11eb-8c5f-33de48705bbe.png">